terraform/README.md

```diff
@@ -3,11 +3,22 @@
 ## Directory structure
 
 * `deployments`: root modules, from where you can run Terraform commands. These
-  should only call the `govuk` composition module and nothing else.
+  should only call the `govuk` composition module, and individual app deployment
+  modules.
```

Comment:
The "These should only call..." sentence is probably more confusing than helpful now that we have two kinds of top-level modules - I'd consider deleting it.